appinventor/components/src/com/google/appinventor/components/runtime/util/GeoJSONUtil.java

```diff
@@ -648,13 +659,15 @@ private void writeMultipolygonGeometryNoHoles(MapPolygon polygon) {
 
     private void writePolygonGeometryNoHoles(MapPolygon polygon) {
       out.print("\"geometry\":{\"type\":\"Polygon\",\"coordinates\":[");
+      out.print("[");
```

Comment:
I'm not sure this addition is correct. Do you have a test to verify that this is needed?

Reply:
This addition is to add extra square bracket in the coordinates section so that the geojson file is parsable as mentioned in the issue.
I have attached the files generated with and without this change by the application.
This is the file generated with this addition
[testgeo.txt](https://github.com/mit-cml/appinventor-sources/files/10398483/testgeo.txt)
This is the file generated without this addition
[testgeo_withoutchange.txt](https://github.com/mit-cml/appinventor-sources/files/10398489/testgeo_withoutchange.txt)
This is the comparison between two files generated
![geojson_comparison](https://user-images.githubusercontent.com/100658781/211987027-234adcf9-3673-49e9-a60d-6e2d9e59b558.png)
The following issue mentioned by the user from the forum is solved
![image](https://user-images.githubusercontent.com/100658781/211987687-1b869a09-662e-42f5-af1f-1a0f577edecc.png)

Reply:
@ewpatton I have added a test for the above changes